lib/tasks/version.rake

```diff
@@ -0,0 +1,23 @@
+namespace :versions do
+  desc 'Sets conference_id in all pre-existing PaperTrail::Version objects'
+  task set_conference_id: :environment do
+
+    PaperTrail::Version.all.each do |version|
+      # All pre-existing versions are either of Conference or Event
+      if version.item_type == 'Conference'
+        version.conference_id = version.item_id
+        version.save
+      elsif version.item_type == 'Event'
+        event = (version.item || version.reify || version.next.reify)
```

Comment:
The task will set the conference_id even if the event has been deleted.It fails only when if the program of that event has been deleted(which is unlikely as there is no interface to do this) as in that case there is no way to find the conference_id using the program_id